tests/features/eds_in_template.feature

```diff
@@ -0,0 +1,33 @@
+# Date:  08/08/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+Feature: Data fetching from template
+
+  Data can be fetched by specifications in the template
+
+  Scenario: Fetch from SPARQL endpoint
+    Given a SPARQL endpoint LOCAL_CORRECT
+    And a SPARQL query SPO_LIMIT_10
```

Comment:
why did you specify this?

Reply:
this is a constant

Reply:
I understand that it's a constant, but why did you need to specify it if the tests don't test against it?